PWGCF/FemtoDream/FemtoDreamCutculator.h

```diff
@@ -47,7 +47,16 @@ class FemtoDreamCutculator
     } catch (const boost::property_tree::ptree_error& e) {
       LOG(fatal) << "Failed to read JSON config file " << configFile << " (" << e.what() << ")";
     }
-    mConfigTree = root.get_child("femto-dream-producer-task");
+
+    // check the config file for all known producer task
+    std::vector<const char*> ProducerTasks = {"femto-dream-producer-task", "femto-dream-producer-reduced-task"};
+    for (auto& Producer : ProducerTasks) {
+      if (root.count(Producer) > 0) {
+        mConfigTree = root.get_child(Producer);
+        std::cout << "Found " << Producer << " in " << configFile << std::endl;
```

Comment:
Please do not use `cout`, use instead `LOGF`

Reply:
fixed.